### ISSUE TITLE
New install. Prompt heads appearing in different color?

### ISSUE THREAD

**Issue by winterhalder** [romkatv/powerlevel10k#576]
Hi,

I've just installed powerline10k to a new Mac, and for some reason the color of the prompt heads doesn't match the rest of the prompt. 

It's minor of course, but still annoying. Thoughts on where my issue may lie? Anybody else have this?

Paul.
<img width="971" alt="2020-03-21_13-53-55" src="https://user-images.githubusercontent.com/3590310/77233173-1aebd000-6b7c-11ea-8da1-6dead06f27e8.png">

**Comment by romkatv** [comment]
There is a setting in iTerm2 that does this but I don't remember what it is. Maybe Minimum Contrast? Try putting different settings (or all of them) to default values.

Also, when you ran the configuration wizard, did you agree to install the font? That step doesn't just install the font but also fixes a dozen problems that users can call upon themselves by tweaking options in iTerm2.

**Comment by winterhalder** [comment]
I did install that font...  

It *is* probably some iTerm2 settings muddling with this. I share those settings with another Mac.

It looks like choosing to use the built-in Powerline glyphs has fixed this - thanks so much.

And thanks for powerline10k - it's awesome!

**Comment by romkatv** [comment]
> It looks like choosing to use the built-in Powerline glyphs has fixed this - thanks so much.

Great!

> And thanks for powerline10k - it's awesome!

You are very welcome 🤗

**Comment by navinthomas** [comment]
@winterhalder Can you explain what you meant by using built-in Powerline glyphs? I want to try out the same thing.

**Comment by navinthomas** [comment]
Using powerline glyphs has not fixed the issue for me. @winterhalder was there anything else you did to fix this issue?

> Also, when you ran the configuration wizard, did you agree to install the font? **That step doesn't just install the font but also fixes a dozen problems that users can call upon themselves by tweaking options in iTerm2.**

@romkatv When I ran the configuration wizard, the setting did not ask me to install the font. I guess it's because I installed the font manually prior to running it. Is there anyway I get the wizard to do those steps for me now?
